var/spack/repos/builtin/packages/r-dupradar/package.py

```diff
@@ -6,17 +6,18 @@
 from spack.package import *
 
 
-class BioconductorDupradar(RPackage):
+class RDupradar(RPackage):
     """Assessment of duplication rates in RNA-Seq datasets"""
 
     homepage = "https://bioconductor.org/packages/3.16/bioc/html/dupRadar.html"
```

Comment:
In all 4 recipes, please get rid of `homepage` and `url`, as it is taken care of by the `bioc` attribute